src/Message/CompletePurchaseResponse.php

```diff
@@ -50,9 +50,8 @@ public function generateSignature()
     public function getCustomFields()
     {
         $fields = array_filter([
-            'Shp_TransactionId' => $this->getTransactionId(),
-            'Shp_Client' => $this->getClient(),
```

Comment:
Why did you remove `Shp_Client`?

Reply:
What is Shp_Client for?

Reply:
Any `Shp_*`is supported by RoboKassa as a custom field.
We use this field in our implementation of integration, but it's optional. It's OK if you don't fill it